taigit/src/libraries/GitHub/src/findBuilds.ts

```diff
@@ -0,0 +1,59 @@
+import axios from 'axios';
+
+export async function
+usesMaven(owner: string, repo: string) : Promise<boolean>{
+    return await fileInRepo(owner, repo, "pom.xml");
+}
+
+export async function
+usesGradle(owner: string, repo: string) : Promise<boolean>{
+    return await fileInRepo(owner, repo, "build.gradle");
+}
+
+export async function
+usesCMake(owner: string, repo: string) : Promise<boolean>{
+    return await fileInRepo(owner, repo, "CMakeLists.txt");
+}
+
+export async function
+usesAnt(owner: string, repo: string) : Promise<boolean>{
+    return await fileInRepo(owner, repo, "build.xml");
+}
+
+export async function 
+usesMake(owner: string, repo: string) : Promise<boolean>{
+    return await fileInRepo(owner, repo, "Makefile");
+}
+
```

Comment:
Please add some javadoc comments here and the next function. The above ones are pretty self explanatory but these should at least have some comments to explain the functionality.